.idea/deploymentTargetDropDown.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This seems to be a personal setting that should not be checked in.
Probably re-evaluate .gitignore rules for `./idea`

Do you want me to do this?

Reply:
go ahead reviewing the rules, but I'll add this to gitignore in the meatime.